src/features/views/components/ViewColumnDialog/choices.tsx

```diff
@@ -35,8 +35,15 @@ export enum CHOICES {
   LOCAL_TEXT = 'localText',
 }
 
+enum COLORS {
+  BLUE = '#1976D2',
+  PURPLE = '#BA68C8',
+  RED = '#ED1C55',
```

Comment:
Guessing it would be better to have these as part of the palette, but we don't have to fix this now.

Reply:
Agree... only way I know how to access the theme is from the useTheme hook - is there an other way that would be possible here you think?

Reply:
i figured it out lol :)

Reply:
I thought I had commented on this already, but for some reason it was never posted. Basically, my suggestion was to have an enum like you had, but instead of having RGB values, it's values should be _names_ of colors from the palette. The component that renders the color would then use `usePalette()` and the name of the color to find the correct color value.

But like I said in my review a moment ago, I don't think this is worth spending time on now.